requirements.txt

```diff
@@ -7,4 +7,5 @@ mysqlclient~=2.0.3                  # https://pypi.org/project/mysqlclient/2.0.3
 pyaml_env~=1.1.1                    # https://github.com/mkaranasou/pyaml_env
 spotipy~=2.19.0                     # https://spotipy.readthedocs.io/en/latest/
 SQLAlchemy~=1.4.26                  # https://pypi.org/project/SQLAlchemy/
-SQLAlchemy-Utils==0.37.8            # https://sqlalchemy-utils.readthedocs.io/
\ No newline at end of file
+SQLAlchemy-Utils==0.37.8            # https://sqlalchemy-utils.readthedocs.io/
+WTForms~=3.0.0
```

Comment:
If you're adding a new dependency, please follow the format and add the comment to the dependency's documentations.

Reply:
This was on the fly, I'll properly format it later